docs/concepts/scheduler.md

```diff
@@ -20,6 +20,21 @@ It should be possible to easily extend and tweak the scheduler in the future.
 Possibly, similar to the Kubernetes scheduler, hooks could be provided which influence the scheduling decisions.
 It should be also possible to completely replace the standard Gardener Scheduler with a custom implementation.
 
+## Algorithm overview
+
+The following **sequence** describes the steps involved to determine a seed candidate:
+
+1. Determine usable seeds (non-deleting, visible)
+1. Consider seeds matching `.spec.seedSelector` in `CloudProfile` used by the `Shoot`
+1. Consider seeds matching `.spec.seedSelector` in `Shoot`
+1. Filter seeds:
+   * that have no network intersection with the `Shoot`'s networks (due to the VPN connectivity between seeds and shoots their networks must be disjoint)
+   * that have `.spec.settings.shootDNS.enabled=false` (only if the shoot specifies a DNS domain or does not use the `unmanaged` DNS provider)
+   * whose taints (`.spec.taints`) are tolerated by the `Shoot` (`.spec.tolerations`)
+   * whose capacity for shoots would be exceeded if the shoot is scheduled onto the seed, see [Ensuring seeds capacity for shoots is not exceeded](#ensuring-seeds-capacity-for-shoots-is-not-exceeded)
+1. Apply active [strategy](#strategies) e.g., _Minimal Distance strategy_
+1. Choose least utilized seed, i.e., the one with the least number of shoot control planes, will be the winner and written to the `.spec.seedName` field of the `Shoot`.
```

Comment:
You can also mention that the `Seed` must be "Ready", i.e., the `Bootstrapped` condition must be `True`.

Reply:
Good idea, thanks.